commons/data_access_layer/cosmos_db.py

```diff
@@ -217,6 +206,16 @@ def attach_context(data: dict, event_context: EventContext):
             "session_id": event_context.session_id,
         }
 
+    @staticmethod
+    def create_sql_date_range_filter(date_range: dict) -> str:
```

Comment:
mm same as query builder. Date range filter only makes sense in time-entries domain. We may consider remove it from here and keep in TimeEntries domain. IF needed of course, otherwise use only time-entries query builder.

Reply:
Ok, adjustments were done